build/root/WORKSPACE

```diff
@@ -33,7 +33,7 @@ load("@io_bazel_rules_go//go:def.bzl", "go_repositories")
 load("@io_bazel_rules_docker//docker:docker.bzl", "docker_repositories")
 
 go_repositories(
-    go_version = "1.8.1",
+    go_version = "1.8.3",
```

Comment:
need to update the rules_go dependency to https://github.com/bazelbuild/rules_go/archive/a9df110cf04e167b33f10473c7e904d780d921e6.tar.gz for this, along with updating the sha256.